openslides_backend/action/actions/projector/project_preview.py

```diff
@@ -0,0 +1,110 @@
+from typing import Any, Dict, List
+
+from ....models.models import Projection, Projector
+from ....services.datastore.commands import GetManyRequest
+from ....shared.filters import And, FilterOperator
+from ....shared.patterns import Collection, FullQualifiedId
+from ...generics.update import UpdateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ...util.typing import ActionData
+from ..projection.set_weight import ProjectionSetWeight
+
+
+@register_action("projector.project_preview")
+class ProjectorProjectPreview(UpdateAction):
+    """
+    Action to get to the next projection.
+    """
+
+    model = Projector()
+    schema = DefaultSchema(Projection()).get_update_schema()
+
+    def get_updated_instances(self, payload: ActionData) -> ActionData:
+        for instance in payload:
+            projection_id = instance.pop("id")
+            projection = self.datastore.get(
+                FullQualifiedId(Collection("projection"), projection_id),
+                ["preview_projector_id"],
+            )
+            # check if projection is from a preview projector
+            if not projection.get("preview_projector_id"):
+                continue
```

Comment:
Do we want to silently ignore if the client sends invalid data @FinnStutzenstein? I would suggest to throw an error here.

Reply:
Yes, an error would be good.

Reply:
Throw an `ActionException` if the `preview_projector_id` is missing.